tests/providers/test_automotive.py

```diff
@@ -146,3 +146,14 @@ def test_fr_FR_plate_format(self):
         plate = self.fake.license_plate()
         assert isinstance(plate, str)
         assert self.pattern.match(plate)
+
+
+class TestNoNO(unittest.TestCase):
+
+    def setUp(self):
+        self.fake = Faker('no_NO')
+        Faker.seed(0)
+
+    def test_sv_SE_plate_format(self):
+        plate = self.fake.license_plate()
+        assert re.match(r"^[A-Z]{2} \d{5}$", plate), "%s is not in the correct format." % plate
```

Comment:
the linting is failing because this file is missing the trailing newline. Could you add a newline at the end of it?

Reply:
@fcurella Fixed in separate commit.